crates/sp-consensus-subspace/src/lib.rs

```diff
@@ -43,6 +43,7 @@ use sp_runtime_interface::{pass_by, runtime_interface};
 #[cfg(feature = "pot")]
 use sp_std::num::NonZeroU32;
 use sp_std::vec::Vec;
+#[cfg(feature = "std")]
```

Comment:
Changes in this file do not belong to the PR and do not even have to be fixed (it is already fixed in one of my PRs about to be merged). It doesn't prevent CI from succeeding, the only reason you have seen it is that compilation of WASM runtime failed. If it didn't fail you wouldn't see it at all.

Reply:
Indeed